go/client/chat_common.go

```diff
@@ -4,13 +4,32 @@
 package client
 
 import (
+	"fmt"
+	"strings"
+
 	"golang.org/x/net/context"
 
 	"github.com/keybase/client/go/libkb"
 	"github.com/keybase/client/go/protocol/keybase1"
+	"github.com/keybase/client/go/teams"
 )
 
 func CheckUserOrTeamName(ctx context.Context, g *libkb.GlobalContext, name string) (*keybase1.UserOrTeamResult, error) {
+	userCli, err := GetUserClient(g)
+	if err != nil {
+		return nil, err
+	}
+	_, err = userCli.LoadUserByName(ctx, keybase1.LoadUserByNameArg{Username: name})
```

Comment:
Why do we do this? The `ComplteAndCanonicalizePrivateTlfName` call should also be checking this for correctness, so why bother with this call?